docs/r2/scalars_and_keras.ipynb

```diff
@@ -307,15 +311,15 @@
       },
       "cell_type": "markdown",
       "source": [
-        "You may see TensorBoard display the message \"No dashboards are active for the current data set\". That's because initial logging data hasn't been saved yet. As training progresses, the Keras model will start logging data. TensorBoard will periodicially refresh and show you your scalar metrics. If you're impatient, you can tap the Refresh arrow on the top right.\n",
+        "You may see TensorBoard display the message \"No dashboards are active for the current data set\". That's because initial logging data hasn't been saved yet. As training progresses, the Keras model will start logging data. TensorBoard will periodicially refresh and show you your scalar metrics. If you're impatient, you can tap the Refresh arrow at the top right.\n",
         "\n",
         "As you watch the training progress, note how both training and validation loss rapidly decrease and then remain stable. In fact, you could have stopped training after 25 epochs, because the training didn't improve much after that point.\n",
         "\n",
         "Hover over the graph to see specific data points. You can also try zooming in with your mouse, or selecting part of them to view more detail.\n",
         "\n",
         "Notice the \"Runs\" selector on the left. A \"run\" represents a set of logs from  a complete round of training, in this case the result of Model.fit(). Developers typically have many, many runs, as they experiment and develop their model over time. \n",
```

Comment:
nit: "Complete round of training" - there can be a run from incomplete rounds of training.

Reply:
Fixed